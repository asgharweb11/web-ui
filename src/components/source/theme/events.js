import theme from "./theme"

export const Color = (item , param) => {
    switch(item){
        case "ligth" :
            return theme.ligth.text.grey;
        case "dark" :
            return theme.dark.text.grey;
    }
}