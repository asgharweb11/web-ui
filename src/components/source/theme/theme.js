export default {
    ligth : {
        main : "white",
        content : "repeating-linear-gradient(0deg , #f2efff , #f3f3ff)",
        primary : "#2196f3",
        secondary : "#ff80ab",
        error : "#f44336",
        warning : "#ff9800",
        info : "#2196f3",
        success : "#4caf50",
        text : {
            grey : "#313131",
            grey1 : "#656565",
            grey2 : "#909090",
            grey3 : "#808080",
            grey4 : "#9a9a9a",
            grey5 : "#c7c7c7",
            color : "#ffffff",
        },
        glass : {
            grey : "#000000a3",
            grey2 : "#00000066",
            grey3 : "#0000003b",
            grey4 : "#0000001c",
            grey5 : "#0000000d",
        },
        hover : {
            primary : "#246fab",
            secondary : "#ec5286",
            error : "#dc1709",
            warning : "#e08703",
            info : "#0d7ad2",
            success : "#237b26",
        }
    },  
    dark : {
        main : "#212121",
        content : "repeating-linear-gradient(0deg ,#333333 ,#333333)",
        primary : "#4dabf5",
        secondary : "#ff99bb",
        error : "#e57373",
        warning : "#ffb74d",
        info : "#64b5f6",
        success : "#81c784",
        text : {
            grey : "#ffffff",
            grey1 : "#ececec",
            grey2 : "#c7c7c7",
            grey3 : "#212121",
            grey4 : "#6b6b6b",
            grey5 : "#464646",
            color : "2b2b2b"
        },
        glass : {
            grey : "#ffffffb3",
            grey2 : "#ffffff87",
            grey3 : "#ffffff59",
            grey4 : "#ffffff2b",
            grey5 : "#ffffff0f",
        },
        hover : {
            primary : "#90ceff",
            secondary : "#ffb5ce",
            error : "#fdaaaa",
            warning : "#ffd393",
            info : "#a8d8ff",
            success : "#b0efb2",
        }

    },
    fontSize : {
        h1 : "40px",
        xl : "25px",
        lg : "20px",
        md : "16px",
        sm : "13px",
        xs : "10px",
    },
    px : {
        p1 : "0px",
        p2 : "1px",
        p3 : "5px",
        p4 : "10px",
        p5 : "15px",
        p6 : "20px",
    },
    media : {
        mobileS: '320px',
        mobileM: '375px',
        mobileL: '425px',
        tablet: '768px',
        tabletL : "991px",
        laptop: '1024px',
        laptopM: '1200px',
        laptopL: '1440px',
        desktop: '2560px'
    }

}