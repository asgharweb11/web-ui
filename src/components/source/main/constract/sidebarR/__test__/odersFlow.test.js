import React from "react"
import renderer from "react-test-renderer"
// -------- Context
import Context from "../../../../../../context.js"
// -------- Styled-ThemeProvider
import {ThemeProvider} from "styled-components"
import theme from "../../../../theme/theme"
//--------- Component
import OdersFlow from "../odersFlow"


test('Component : Constract/sidebarR/odersFlow.js', () => {
    const container = renderer.create(<Context>
        <ThemeProvider theme={theme}>
            <OdersFlow />
        </ThemeProvider>
    </Context>)

    let tree = container.toJSON();
    expect(tree).toMatchSnapshot()
    expect(tree.children[1].children[0].children[0].children[0].children[0]).toEqual("B")

})
