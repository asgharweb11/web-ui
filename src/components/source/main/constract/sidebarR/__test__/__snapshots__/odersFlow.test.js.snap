// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component : Constract/sidebarR/odersFlow.js 1`] = `
<div
  className="sc-bdfBwQ deFIOY"
>
  <div
    className="title"
  >
    Oders
    <span>
      Flow
    </span>
  </div>
  <div
    className="allCarda"
  >
    <div
      className="cardM"
    >
      <div
        className="header"
      >
        <div
          className="logo"
        >
          B
        </div>
        <div
          className="users"
        >
          <div
            className="user"
          >
            <img
              alt="user"
              src="/files/pic/user/3.png"
            />
          </div>
          <div
            className="user"
          >
            <img
              alt="user"
              src="/files/pic/user/4.png"
            />
          </div>
          <div
            className="user"
          >
            <img
              alt="user"
              src="/files/pic/user/1.png"
            />
          </div>
        </div>
      </div>
      <div
        className="body"
      >
        <div
          className="location"
        >
          <h3>
            South Brookly
          </h3>
          <p>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-map-marked fa-w-18 "
              data-icon="map-marked"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M288 0c-69.59 0-126 56.41-126 126 0 56.26 82.35 158.8 113.9 196.02 6.39 7.54 17.82 7.54 24.2 0C331.65 284.8 414 182.26 414 126 414 56.41 357.59 0 288 0zM20.12 215.95A32.006 32.006 0 0 0 0 245.66v250.32c0 11.32 11.43 19.06 21.94 14.86L160 448V214.92c-8.84-15.98-16.07-31.54-21.25-46.42L20.12 215.95zM288 359.67c-14.07 0-27.38-6.18-36.51-16.96-19.66-23.2-40.57-49.62-59.49-76.72v182l192 64V266c-18.92 27.09-39.82 53.52-59.49 76.72-9.13 10.77-22.44 16.95-36.51 16.95zm266.06-198.51L416 224v288l139.88-55.95A31.996 31.996 0 0 0 576 426.34V176.02c0-11.32-11.43-19.06-21.94-14.86z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
             1945 frederick street
          </p>
        </div>
        <button
          className="btn btn-warning btn-sm"
          disabled={false}
          type="button"
        >
          ON THE WAY
        </button>
      </div>
    </div>
    <div
      className="cardM"
    >
      <div
        className="header"
      >
        <div
          className="logo"
        >
          B
        </div>
        <div
          className="users"
        >
          <div
            className="user"
          >
            <img
              alt="user"
              src="/files/pic/user/3.png"
            />
          </div>
          <div
            className="user"
          >
            <img
              alt="user"
              src="/files/pic/user/4.png"
            />
          </div>
          <div
            className="user"
          >
            <img
              alt="user"
              src="/files/pic/user/1.png"
            />
          </div>
        </div>
      </div>
      <div
        className="body"
      >
        <div
          className="location"
        >
          <h3>
            South Brookly
          </h3>
          <p>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-map-marked fa-w-18 "
              data-icon="map-marked"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M288 0c-69.59 0-126 56.41-126 126 0 56.26 82.35 158.8 113.9 196.02 6.39 7.54 17.82 7.54 24.2 0C331.65 284.8 414 182.26 414 126 414 56.41 357.59 0 288 0zM20.12 215.95A32.006 32.006 0 0 0 0 245.66v250.32c0 11.32 11.43 19.06 21.94 14.86L160 448V214.92c-8.84-15.98-16.07-31.54-21.25-46.42L20.12 215.95zM288 359.67c-14.07 0-27.38-6.18-36.51-16.96-19.66-23.2-40.57-49.62-59.49-76.72v182l192 64V266c-18.92 27.09-39.82 53.52-59.49 76.72-9.13 10.77-22.44 16.95-36.51 16.95zm266.06-198.51L416 224v288l139.88-55.95A31.996 31.996 0 0 0 576 426.34V176.02c0-11.32-11.43-19.06-21.94-14.86z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
             1945 frederick street
          </p>
        </div>
        <button
          className="btn btn-warning btn-sm"
          disabled={false}
          type="button"
        >
          ON THE WAY
        </button>
      </div>
    </div>
  </div>
</div>
`;
